languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10084,6 +10084,378 @@ though, and has since been slightly modified)-->
+              <antipattern>
+                  <token>être</token>
+                  <token>soit</token>
+                  <token max="1" postag="M fin" spacebefore="no">.</token>
+              </antipattern>
+              <pattern>
+                  <marker>
+                      <token postag="V.*|R dem m s|A.*|J.*" postag_regexp="yes"><exception regexp="yes">ne|ce|ça|cela|alors|bien</exception></token>
+                      <token>soit</token>
+                  </marker>
+                  <token postag="M fin" spacebefore="no">.<exception scope="next">.</exception></token>
+              </pattern>
+              <message>Une virgule semble être requise dans ce contexte.</message>
+              <suggestion>\1, \2</suggestion>
+              <example correction="arrive, soit">Il <marker>arrive soit</marker>.</example>
+              <example correction="plus, soit">Il ne le voit <marker>plus soit</marker>.</example>
+              <example>Il ne le voit <marker>plus soit</marker>...</example>
+              <example>Il le faut, <marker>alors soit</marker>.</example>
+              <example>Il n'a de conflit avec qui que <marker>cela soit</marker>.</example>
+          </rule>
+          <rule>
+              <antipattern>
+                  <token postag="SENT_START"/>
+                  <token>soit</token>
+                  <token regexp="yes" skip="-1">[^,;:]+</token>
+                  <token>soit</token>
+              </antipattern>
+              <pattern>
+                  <token postag="SENT_START"/>
+                  <marker>
+                      <token>soit</token>
+                      <token regexp="yes">[^,);:(«»-]+<exception regexp="yes">[a-z]$|[A-Z]$</exception></token></marker>
+              </pattern>
+              <message>Une virgule est conseillée dans ce contexte.</message>
+              <suggestion>\2, \3</suggestion>
+              <example correction="Soit, il"><marker>Soit il</marker> peut venir avec nous.</example>
+              <example correction="Soit, prendre"><marker>Soit prendre</marker> la vie côté plaisir est un concept intéressant.</example>
+              <example correction="Soit, grand"><marker>Soit grand</marker> ou petit, peu m'importe.</example>
+              <example><marker>Soit V</marker> le centre du cercle.</example>
+              <example><marker>Soit grand</marker> soit petit, peu m'importe.</example>
+          </rule>
+          <rule>
+              <pattern>
+                  <token skip="-1">soit</token>
+                  <marker>
+                      <token postag="N.*" postag_regexp="yes"/>
+                      <token>soit</token>
+                  </marker>
+              </pattern>
+              <message>Une virgule est conseillée dans ce contexte.</message>
+              <suggestion>\2, \3</suggestion>
+              <example correction="bus, soit">Je dois soit prendre le <marker>bus soit</marker> le train.</example>
+          </rule>
+          <rule>
```

Comment:
Only few FPs are left:

> Et si tu fais manger une famille avec 50€ soit 60CHF, c'**est soit** que ta famille c'est une seule personne, soit que vous mangez pas vraiment pas beaucoup.

Reply:
Okay, I'll stop reviewing here — you can go ahead and extend the special characters (>" etc.) and review the rule editor results. Please let me know in a comment and/or on Slack when the rule is ready (i.e. when there are no FP in the rule editor) or when there are any questions or problems.